### PR TITLE
Sync Payment Methods in Budget Section with Dashboard

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/4453

Peek 2021-07-15 14-13

### CODE REVIEW

**Review by kewitz — APPROVED** [review]
Nice work! I suspected was something like this.

**Comment by SudharakaP** [comment]
> Nice work! I suspected was something like this.

@kewitz : Thanks. Could you review it again please, I changed some code as it seemed some tests were failing. Thing is that we were using that query in some other components. So I've isolated that query and created some new queries in those components instead. 😸

File: components/collective-page/graphql/preload.js [hunk 1]
```diff
@@ -16,6 +18,22 @@ import { getUpdatesSectionQueryVariables, updatesSectionQuery } from '../section
...
+      totalCount
+      nodes {
+        ...ExpensesListFieldsFragment
+      }
+    }
+  }
+  ${transactionsQueryCollectionFragment}
+  ${expensesListFieldsFragment}
+`;
+
```
Comment (kewitz) [inline components/collective-page/graphql/preload.js:36]: Move this to `./queries`.

File: components/host-dashboard/AddFundsModal.js [hunk 2]
```diff
@@ -109,6 +111,22 @@ const addPlatformTipMutation = gqlV2/* GraphQL */ `
...
+      totalCount
+      nodes {
+        ...ExpensesListFieldsFragment
+      }
+    }
+  }
+  ${transactionsQueryCollectionFragment}
+  ${expensesListFieldsFragment}
+`;
+
```
Comment (kewitz) [inline components/host-dashboard/AddFundsModal.js:129]: Duplicated, import this from `collective-page/graphql/queries.js` after you move the other one.

File: components/collective-page/graphql/preload.js (hunk 1 above)
Comment (SudharakaP) [reply components/collective-page/graphql/preload.js]: @kewitz : Thanks. Corrected that. 🙂

File: components/host-dashboard/AddFundsModal.js (hunk 2 above)
Comment (SudharakaP) [reply components/host-dashboard/AddFundsModal.js]: Corrected this. Thanks. 👍🏽

**Pull request merged**